packages/schema/src/dialects/mssql.ts

```diff
@@ -15,8 +15,10 @@ export default class MSSQL extends KnexMSSQL implements SchemaInspector {
 				c.IS_NULLABLE as is_nullable,
 				c.DATA_TYPE as data_type,
 				c.CHARACTER_MAXIMUM_LENGTH as max_length,
+       			cc.is_computed as is_generated,
```

Comment:
cc doesn't exist here

Reply:
Oh. This has to be removed, it was from another solution I tried.